app/serializers/user_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class UserSerializer < ActiveModel::Serializer
-  attributes :id, :first_name, :last_name, :email, :phone, :password, :role, :organization
+  attributes :id, :first_name, :last_name, :email, :phone, :password, :role, :organization, :events
```

Comment:
why events are serialized for user? do we always need them? it will be serialized even when we access user profile